src/pkg/packaging/deb/package.targets

```diff
@@ -9,6 +9,11 @@
                                     $(PackageTargetRid.StartsWith('debian'))">true</IsDebianBasedDistro>
     <BuildDebPackage Condition="'$(IsDebianBasedDistro)' == true and '$(TargetArchitecture)' == 'x64'">true</BuildDebPackage>
     <DebianConfigJsonName>debian_config.json</DebianConfigJsonName>
+    <RuntimeDebsDependsOn>
```

Comment:
Why split this into a separate property?  I don't think this provides any value over just putting the DependsOnTargets directly into the Target definition below.  This just adds a level of indirection that the reader has to go look for.